### PR TITLE
[FIX] *: rephrase, correct typos

### PR DESCRIPTION
**Pull request by mart-e**
Courtesy of Transifex's translators for reporting bad/unclear sentences.

### CODE REVIEW

**Comment by mart-e** [comment]
@robodoo r+

File: addons/project/models/project.py [hunk 1]
```diff
@@ -626,7 +626,7 @@ def _read_group_stage_ids(self, stages, domain, order):
     parent_id = fields.Many2one('project.task', string='Parent Task', index=True)
     child_ids = fields.One2many('project.task', 'parent_id', string="Sub-tasks", context={'active_test': False})
     subtask_project_id = fields.Many2one('project.project', related="project_id.subtask_project_id", string='Sub-task Project', readonly=True)
-    allow_subtasks = fields.Boolean('project.project', related="project_id.allow_subtasks", readonly=True)
+    allow_subtasks = fields.Boolean(string="Allow Sub-tasks", related="project_id.allow_subtasks", readonly=True)
```
Comment (Feyensv) [inline addons/project/models/project.py:629]: :rofl:

File: addons/purchase/models/res_config_settings.py [hunk 2]
```diff
@@ -30,7 +30,8 @@ class ResConfigSettings(models.TransientModel):
         config_parameter='purchase.use_po_lead',
         help="Margin of error for vendor lead times. When the system generates Purchase Orders for reordering products,they will be scheduled that many days earlier to cope with unexpected vendor delays.")
 
-    group_send_reminder = fields.Boolean("Receipt Reminder", implied_group='purchase.group_send_reminder', default=True, help="Allow automatically send email to reminder your vendor the receipt date")
+    group_send_reminder = fields.Boolean("Receipt Reminder", implied_group='purchase.group_send_reminder', default=True,
+        help="Allow automatically send email to reminde your vendor the receipt date")
```
Comment (Feyensv) [inline addons/purchase/models/res_config_settings.py:34]: **reminde** ?

File: addons/sms/wizard/sms_composer.py [hunk 3]
```diff
@@ -41,15 +41,15 @@ def default_get(self, fields):
     res_ids = fields.Char('Document IDs')
     res_ids_count = fields.Integer(
         'Visible records count', compute='_compute_recipients_count', compute_sudo=False,
-        help='UX field computing the number of recipients in mass mode without active domain')
+        help='Number of recipients that will recieve the SMS if sent in mass mode, without applying the Active Domain value')
```
Comment (Feyensv) [inline addons/sms/wizard/sms_composer.py:44]: **receive** ;)

File: addons/purchase/models/res_config_settings.py (hunk 2 above)
Comment (mart-e) [reply addons/purchase/models/res_config_settings.py:34]: One letter at a time !

**Pull request closed without merging**